pkg/integrations/ssl_exporter/ssl_exporter.go

```diff
@@ -0,0 +1,128 @@
+// Package ssl_exporter embeds https://github.com/ribbybibby/ssl_exporter/v2
+package ssl_exporter
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+
+	"github.com/go-kit/log"
+	"github.com/grafana/agent/pkg/integrations"
+	"github.com/grafana/agent/pkg/integrations/config"
+	ssl_config "github.com/ribbybibby/ssl_exporter/v2/config"
+)
+
+// DefaultConfig holds the default settings for the ssl_exporter integration.
+var DefaultConfig = Config{
+	ConfigFile: "",
+	SSLTargets: []SSLTarget{},
+}
+
+// SSLTarget represents a target to scrape.
+type SSLTarget struct {
+	Name   string `yaml:"name"`
+	Target string `yaml:"target"`
+	Module string `yaml:"module"`
+}
+
+// Config controls the ssl_exporter integration.
+type Config struct {
+	ConfigFile string      `yaml:"key_file,omitempty"`
```

Comment:
In the documentation we're calling this `config_file`; let's settle on one of two names. Also, we might want to mention whether it should always be an absolute path (depending on how the upstream LoadConfig methods uses this field)

Reply:
not sure why that was key_file, changed it to config_file